templates/user/account_delete_form.html

```diff
@@ -0,0 +1,70 @@
+{% extends "base.html" %}
+
+{% block title %}My user profile{% endblock title %}
+
+{% block content %}
+<!-- Begin Page Content -->
+<div class="container-sm py-5">
+    <!-- Main Content Here -->
+    <div class="card border-0 shadow p-4">
```

Comment:
This padding looks weird. Add it in the body instead
```suggestion
    <div class="card border-0 shadow">
```

Reply:
Yes, will do